accounts-db/src/accounts_db.rs

```diff
@@ -3016,6 +3053,7 @@ impl AccountsDb {
         self.clean_accounts_stats.report();
         datapoint_info!(
             "clean_accounts",
+            ("slot", max_clean_root_inclusive.unwrap_or_default(), i64),
```

Comment:
this slot metric could be pulled out. There are a few lines due to this. It is a totally separate idea. That could simplify some.